frontend/entrypoint.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+
+HTML=/usr/share/nginx/html/index.html
+
+for var in $(env | grep '^VITE_' | cut -d= -f1); do
+  placeholder="__${var}__"
+  sed -i "s|${placeholder}|${!var}|g" "$HTML"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, indirect expansion is undefined.

(SC3053)